cfn-resources/cloud-backup-snapshot-export-job/test/cfn-test-delete-inputs.sh

```diff
@@ -42,3 +43,35 @@ echo "--------------------------------delete role starts -----------------------
 aws iam delete-role-policy --role-name "$roleName" --policy-name "$policyName"
 aws iam delete-role --role-name "$roleName"
 echo "--------------------------------delete role ends ----------------------------"
+
+
+# delete cluster
+if atlas clusters delete "$clusterName" --projectId "${projectId}" --force
+then
+    echo "$clusterName cluster deletion OK"
+else
+    (echo "Failed cleaning cluster:$clusterName" && exit 1)
+fi
+
+echo "Waiting for cluster to get deleted"
+
+status=$(atlas clusters describe "${clusterName}" --projectId "${projectId}" --output=json | jq -r '.stateName')
+echo "status: ${status}"
+
+while atlas clusters describe "${clusterName}" --projectId "${projectId}"; do
+        sleep 30
+        if atlas clusters describe "${clusterName}" --projectId "${projectId}"
+        then
+          status=$(atlas clusters describe "${clusterName}" --projectId "${projectId}"  --output=json | jq -r '.stateName')
+        else
+          status="DELETED"
+        fi
+        echo "status: ${status}"
+done
```

Comment:
unfortunately, `atlas cluster describe` will return a 404 error once the cluster is deleted. There is no `DELETED` status

Reply:
@andreaangiolillo  atlas clusters describe command will become false in WHILE loop and comes out. This has been tested before and working I believe. 
Any thoughts on using other commands just like WATCH here...

Reply:
Thanks for the clarification. No, the watch command won't work here 😢